src/components/Book.js

```diff
@@ -0,0 +1,26 @@
+import './Book.css';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({ book }) {
```

Comment:
- [ ] It seems that most of the rendering is gone since your last commit.  This includes almost everything, like the form:

![image](https://user-images.githubusercontent.com/57421823/160070112-42ab0f7b-db4e-466b-a235-fb02b34319a4.png)

It seems that the reason is that your props are not working well.  Kindly fix this bug so the project works as before, while using props.

Reply:
The props are working as expected, I just forget to add the input form after the book list,
Thank you for your review 👍🏼